bigchaindb/pipelines/stale.py

```diff
@@ -71,6 +71,5 @@ def start(timeout=5, backlog_reassign_delay=5):
     """Create, start, and return the block pipeline."""
     pipeline = create_pipeline(timeout=timeout,
                                backlog_reassign_delay=backlog_reassign_delay)
-    pipeline.setup()
```

Comment:
`setup` prepares the pipeline to be used and it's called in `__init__`. You can call `setup` again if you want to reconfigure the pipeline. Calling `setup` with no args doesn't do anything.